Doc/c-api/typeobj.rst

```diff
@@ -1177,6 +1177,17 @@ and :c:type:`PyType_Type` effectively act as defaults.)
 
       .. versionadded:: 3.10
 
+   .. data:: Py_TPFLAGS_IMMUTABLE
```

Comment:
I honestly still think the flag should be named IMMUTABLETYPE, since there are plenty of flags in tp_flags that say something about the instances, and "has immutable instances" is just a likely enough inpterpretation of an IMMUTABLE flag that it might be confusing.

Reply:
Fixed in ef26fb7bbba50e3f3c665199ba904f84229587ec